src/templates/tournaments/tournament_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete Tournament</h1>
+  <p>Are you sure you want to delete the tournament "{{ tournament.name }}"?</p>
+  <form method="post">
+    {% csrf_token %}
+    <button type="submit">Yes, delete</button>
```

Comment:
It would be nice to have a no button, but for the purposes of this PR, this is good for now.

Reply:
Thanks!